docs/advanced/debugging.rst

```diff
@@ -27,15 +27,335 @@ And, of course, if you're hitting that ever-challenging ``No scope with a Tag ma
 Symbols and Sources
```

Comment:
It feels like the Symbols and Sources stuff should maybe go after the Diagnostics bits? Like, stepping in to Autofac source should only happen after you've tried all the diagnostics investigation?

Reply:
That makes sense. I can reorder.